release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -444,6 +444,39 @@ For more information, see the following sections:
 [id="ocp-4-15-storage"]
 === Storage
 
+[id="ocp-4.15-storage-recovering-vgs-from-prev-installation"]
+==== Recovering volume groups from the previous {lvms} installation
+With this release, `LVMCluster` custom resource (CR) provides support for recovering volume groups from the previous {lvms} installation. If the `deviceClasses.name` field is set to the name of a volume group from the previous {lvms} installation, {lvms} recreates that volume group in the current {lvms} installation. This simplifies the process of data recovery through the re-installation of {lvms}.
```

Comment:
```suggestion
With this release, `LVMCluster` custom resource (CR) provides support for recovering volume groups from the previous {lvms} installation. If the `deviceClasses.name` field is set to the name of a volume group from the previous {lvms} installation, {lvms} recreates resources related to that volume group in the current {lvms} installation. This simplifies the process of using devices from previous installations through the re-installation of {lvms}.
```
This does not allow data recovery but convenience to the users to utilize existing devices without the need to wipe them.

Reply:
Updated as per the suggestion.